### PR TITLE
Fixes word

### PR DESCRIPTION
**Pull request by FloranOtten**
:cl:
spellcheck: Deadminned themselves. 
/:cl:

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -633,9 +633,10 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 
 	holder.deactivate()
 
-	to_chat(src, "<span class='interface'>You are now a normal player.</span>", confidential = TRUE)
-	log_admin("[src] deadmined themself.")
-	message_admins("[src] deadmined themself.")
+
+	to_chat(src, "<span class='interface'>You are now a normal player.</span>")
+	log_admin("[src] deadmined themselves.")
```
Comment (Fikou) [inline code/modules/admin/admin_verbs.dm:638]: shouldnt it be deadminned

Comment (FloranOtten) [reply code/modules/admin/admin_verbs.dm:638]: Didn't notice, I'll add that to the PR

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -633,9 +633,10 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 
 	holder.deactivate()
 
-	to_chat(src, "<span class='interface'>You are now a normal player.</span>", confidential = TRUE)
-	log_admin("[src] deadmined themself.")
-	message_admins("[src] deadmined themself.")
+
+	to_chat(src, "<span class='interface'>You are now a normal player.</span>")
+	log_admin("[src] deadminned themselves.")
```
Comment (YPOQ) [inline code/modules/admin/admin_verbs.dm:638]: Why change it to a plural pronoun when it's referring to one person?

Comment (optimumtact) [inline code/modules/admin/admin_verbs.dm:638]:
```suggestion
	log_admin("[src] deadminned themself.")
```

Comment (FloranOtten) [reply code/modules/admin/admin_verbs.dm:638]: 'cause there's no gender-neutral singular pronoun. It's them or nothing.

**Pull request merged**